### PR TITLE
[55517] Implemented timed disable/enable of back buttons to prevent them being read first by voiceover

### PR DESCRIPTION
**Pull request by jonasofrich**

### CODE REVIEW

**Review by pavshr — CHANGES_REQUESTED** [review]
I would refactor code a bit now, since it is called at so many screens:

1. rename `removeAccessibilityElementAndEnableAfterDelay()` to `postAccessibilityNotificationAndReenableElement()`
2. pass the UIView you want to post notification to as a parameter to `postAccessibilityNotificationAndReenableElement` so ` UIAccessibility.PostNotification(UIAccessibilityPostNotification.LayoutChanged,...)` is called from there
3. add a VoiceOver check to `postAccessibilityNotificationAndReenableElement` to it gets called only when VO is on
4. refactor the calls

**Comment by jonasofrich** [comment]
@pavshr like this?

File: NDB.Covid19/NDB.Covid19.iOS/Views/Welcome/ChildViews/PageViewController.cs [hunk 1]
```diff
@@ -50,5 +51,27 @@ protected void InitSubTitle(UILabel label, string text)
             InitLabel(label, FontType.FontBold, text, 16, 22);
             label.AccessibilityAttributedLabel = AccessibilityUtils.RemovePoorlySpokenSymbols(text);
         }
+
+        /// <summary>
```
Comment (pavshr) [inline NDB.Covid19/NDB.Covid19.iOS/Views/Welcome/ChildViews/PageViewController.cs:55]: Interesting, did you just try to set it as subclass to BaseViewController?

**Review by pavshr — CHANGES_REQUESTED** [review]

File: NDB.Covid19/NDB.Covid19.iOS/Views/Welcome/ChildViews/PageViewController.cs (hunk 1 above)
Comment (jonasofrich) [reply NDB.Covid19/NDB.Covid19.iOS/Views/Welcome/ChildViews/PageViewController.cs]: No, didn't think of that, but that is better. Seems to behave the same way. Fixed now.

**Review by pavshr — APPROVED** [review]

**Pull request merged**